v_3_1_1/types.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	defaultHyperkubeApiserverBindAddress = "${DEFAULT_IPV4}"
+	defaultHyperkubeApiserverBindAddress = "$(HOST_IP)"
```

Comment:
Unrelated: What is `$(HOST_IP)`? What are parenthesis doing?

Reply:
It is env variable, which comes from pod.IP https://github.com/giantswarm/k8scloudconfig/blob/e0bb4328626571fb87adfa9ddb6380b9eb1fc63e/v_3_1_1/master_template.go#L1792
Parenthesis is the way to access env vars as arguments for container in pod

Reply:
Why it isn't `${HOST_IP}`?

Reply:
Because it is the way to distinguish way of accessing env vars. You can't pass host env vars with that simple reference. If you could - all those changes would be useless. 
Here is the example: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#use-environment-variables-to-define-arguments

Reply:
Thanks for explaining that.

Reply:
please add the same explanation to this constant. very specific things.

I had exactly the same questions as Pawel :)